packages/node_modules/@webex/webex-core/src/credentials-config.js

```diff
@@ -32,6 +32,18 @@ const CredentialsConfig = AmpState.extend({
       */
     authorizationString: ['string', false, process.env.WEBEX_AUTHORIZATION_STRING || process.env.AUTHORIZATION_STRING],
 
+    /**
+     * Authorization URL which prompts for user's password. Inferred from
+     * {@link config.credentials.authorizationString}.
+     * @type {string}
+     */
+    authorizeUrl: [
+      'string',
+      false,
+      process.env.AUTHORIZE_URL ||
```

Comment:
Since this isn't used or documented anywhere, could we prefix it with `WEBEX_` to try to get our env vars prefixed?
```suggestion
      process.env.WEBEX_AUTHORIZE_URL ||
```

Reply:
Yep, can do.